libmscore/style.h

```diff
@@ -1096,6 +1096,94 @@ enum class Sid {
       STYLES
       };
 
+// Internal resolution factor that multiplies values originally in points
+static constexpr qreal DPI_F = 5; // screen resolution factor
+
+// Unit conversion factors
+static constexpr qreal INCH  = 25.4;  // millimeters per inch
```

Comment:
These 3 variable names stink.  They should be renamed.  `INCH` is the one that pre-dates this PR.  It's the one that is used the most in the code.  The other 2 are less important.

I'd like to do it here, but you probably want it in a separate PR.  Here is the change I would propose:
Rename `INCH` to `MMPI` which stands for millimeters per inch.  That is consistent with the variable names directly below.

As for the other two names, that's uglier.  Both refer to units that start with the same letter as another, more common unit.  I don't mind leaving them the same as they are now.  Otherwise I could:
Rename 'PICA' to 'PPP' or 'PIPP' or `PIPT` or `PICA2POINT` ...those names all stink too
Rename 'DIDOT' to 'DPP' or 'DIPP' or 'DIPPT' or 'DIDOT2POINT' ...again, not so great.
Fortunately these two constants are only used within this .h file and in pagesettings.cpp.  So they are less important.  `INCH` is used in many places and is totally confusing as a name.